### DISCUSSION TITLE
I can't obtain the pa.json[.gz] file to create plots using exportPlots

### DISCUSSION THREAD

**Discussion by DiegoDMaldonado** [milaboratory/mixcr#1521] [Q&A] [answered]
I do the RNA-seq analysis and use SAM file,I obtained the expected output files with that analysis. However, during the process, I realized that I do not have the required pa.json[.gz] file to create the plots (mixcr exportPlots) for my analysis.

Could someone guide me on how to create the pa.json[.gz] file or provide any insights on obtaining it? I am particularly interested in generating plots from the analysis, and having the pa.json[.gz] file is crucial for this task.

I read the documentation, but I can't find how create that file.

I use this for my analysis, the MiXCR v4.3.2., with this code : java -jar mixcr.jar analyze rnaseq-bcr-full-length -s mmu Immunization.bam results --verbose

Any assistance, suggestions, or examples related to creating the pa.json[.gz] file would be greatly appreciated.

Thank you for your time and assistance.

**Comment by mizraelson** [comment, marked as answer]
Hi,
This file is generated by [`mixcr postanalysis individual`](https://mixcr.com/mixcr/reference/mixcr-postanalysis/#individual-postanalysis) function. As an input you should a use `.clns` file (or files in case of multiple samples) generated by `mixcr analyze`.
